spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/integration/inbound/pull/RocketMQAckCallback.java

```diff
@@ -79,7 +81,7 @@ public void acknowledge(Status status) {
 				switch (status) {
 				case REJECT:
 				case ACCEPT:
-					consumer.committed(messageQueue);
+					consumer.commit(Collections.singleton(messageQueue), false);
```

Comment:
We need to evaluate whether it is appropriate to set the `persist` parameter here to `false`.